params/cluster.go

```diff
@@ -9,11 +9,11 @@ type cluster struct {
 var ropstenCluster = cluster{
 	NetworkID: 3,
 	BootNodes: []string{
-		"enode://8472a9a236afe091e9941909a23b8e987287ad76839da2d9260b3d1697fea180d494617d42e605a5c229c7bf326e2bfa3d1a390fd03bf1783376c275e380126e@206.189.108.52:30404", // boot-01.do-ams3.eth.beta
-		"enode://c266882060e6670030fac53f07dfd802d51441c72b188722352d964095ab79f6aa1350b9000a2ab3a2d76a835c79ffcf6cd022a656ec04e7a02683417364ce5e@128.199.55.181:30404", // boot-02.do-ams3.eth.beta
-		"enode://f9c7d50832fd42a15157f4150b1ce3d306b8dc192fe4e3dbeb1761c8913a61fc62ed752235ccf5759c57b9b1c3634e57ad19e5d9e13b79f317f4ef16e6847d9b@35.202.99.224:30404",  // boot-01.gc-us-central1-a.eth.beta
-		"enode://e81a60977e5e420c8dcdda4b5d2b419f629dd9f630f6db71daaf84364927c341db6d4fa9c509ebebb9a5cd967b2c2ed9f67489dae3ba17d5e4edf71a13b3be6b@104.154.154.58:30404", // boot-02.gc-us-central1-a.eth.beta
-		"enode://a7a0e0961b971db2b217bb494c844fa39ca8a3422537e07451265e55208ec757f6c3a13b888169c8b14f584ab3bda1fd134cf316f236b61a5b719138bc42e801@185.156.42.129:30404", // boot-03.eth.beta
+		"enode://436cc6f674928fdc9a9f7990f2944002b685d1c37f025c1be425185b5b1f0900feaf1ccc2a6130268f9901be4a7d252f37302c8335a2c1a62736e9232691cc3a@174.138.105.243:30404", // boot-01.do-ams3.eth.beta
+		"enode://5395aab7833f1ecb671b59bf0521cf20224fe8162fc3d2675de4ee4d5636a75ec32d13268fc184df8d1ddfa803943906882da62a4df42d4fccf6d17808156a87@206.189.243.57:30404",  // boot-02.do-ams3.eth.beta
+		"enode://f9c7d50832fd42a15157f4150b1ce3d306b8dc192fe4e3dbeb1761c8913a61fc62ed752235ccf5759c57b9b1c3634e57ad19e5d9e13b79f317f4ef16e6847d9b@35.202.99.224:30404",   // boot-01.gc-us-central1-a.eth.beta
+		"enode://e81a60977e5e420c8dcdda4b5d2b419f629dd9f630f6db71daaf84364927c341db6d4fa9c509ebebb9a5cd967b2c2ed9f67489dae3ba17d5e4edf71a13b3be6b@104.154.154.58:30404",  // boot-02.gc-us-central1-a.eth.beta
```

Comment:
This one in GCE does not exist here: http://consul.status.im/ui/gc-us-central1-a/services/statusd-boot